src/components/game/index.jsx

```diff
@@ -1,9 +1,71 @@
-import React from "react";
+import React, {useState} from "react";
 import Board from "components/game/Board";
 
 // Renders the Board component
 function Game() {
-  return <Board />;
+  const [valuesArr, setValuesArr] = useState(Array(9).fill(null)); // Values of the 1st-9th square
+  const [nextPlayer, setNextPlayer] = useState("O"); // Initial player aka first player to go is O
+
+  let winner = checkWinner(valuesArr);
```

Comment:
You can turn `winner` into a state and then change it during `updateValuesArr` instead of computing this every time the component rerenders. This will cause performance issues if we have a huge board, in this project it's fine.